### PR TITLE
Additional units   exp zom, juk zom, air zom, potato mine, air monkey 

### PR DESCRIPTION
**Pull request by Kirmito**

### CODE REVIEW

**Comment by dershao** [comment]
Can you pull from master branch, solve merge conflicts, then re-push your changes to the branch?

**Comment by Kirmito** [comment]
sure

File: bin/.gitignore [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/input/
```
Comment (dershao) [inline bin/.gitignore]: Delete this and all images found in /bin folder

**Comment by Kirmito** [comment]
Ok

File: src/assets/Air_Monkey.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		return PLANT_TYPE;
+	}
+
+	@Override
+	public void attack(Board board) {
+		Grid[][] gameBoard = board.getBoard();
+		int row = getRow();
+		int column = getCol();
+		
+		for (int col = column; col < gameBoard[row].length; col++) {
```
Comment (dershao) [inline src/assets/Air_Monkey.java]: Use the new board `getFirstZombie()` method

**Pull request merged**